client/public/bundle.js.LICENSE.txt

```diff
@@ -1,3 +1,105 @@
+/**
```

Comment:
We can probably remove this file, I don't think it is adding much to the code base

Reply:
Agreed. I think (hope) I removed it properly in the commit that responds to your requests.